infrastructure/output.tf

```diff
@@ -40,4 +40,8 @@ output "test_encryption_enabled" {
   value = "${var.encyption_enabled}"
 }
 
+output "test_failure_email" {
+  value = "ignore@failure.email"
```

Comment:
What's this for?

Reply:
to be used in preview prs. so it goes _smoothly_. temporary value anyway - no need for alerts actually show up in preview

Reply:
but its an output? where is it used?

Reply:
i thought jenkinsfile uses these values when `loadVaultSecrets`

Reply:
its outputted for tests, its only there for smoke and functional tests